consensus/broadcast/commonMessenger.go

```diff
@@ -93,6 +81,9 @@ func checkCommonMessengerNilParameters(
 	if check.IfNil(args.KeysHandler) {
 		return ErrNilKeysHandler
 	}
+	if check.IfNil(args.DelayedBroadcaster) {
```

Comment:
unit tests should be added for this case as well, see this test for example `TestMetaChainMessenger_NilKeysHandlerShouldError`, with its equivalent on shard `TestShardChainMessenger_NilKeysHandlerShouldError`

Reply:
updated